pkg/pillar/hypervisor/kvm.go

```diff
@@ -635,12 +660,18 @@ func (ctx kvmContext) GetDomsCPUMem() (map[string]types.DomainMetric, error) {
 		if err != nil {
 			return res, logError("can't access stats for domain %s PID %d (%v)", dom, pid, err)
 		}
+		usedMemory := uint32(roundFromBytesToMbytes(uint64(s.ResidentMemory())))
+		availableMemory := uint32(roundFromBytesToMbytes(uint64(s.VirtualMemory())))
+		usedMemoryPerc := float64(0.0)
```

Comment:
Does gopsutil's memory function return the same thing? I'm trying to have less variation in how we retrieve stats from Linux.

Reply:
moved to gopsutil.